test/suite/test_reconfig01.py

```diff
@@ -95,19 +95,24 @@ def test_reconfig_checkpoints(self):
         self.conn.reconfigure("checkpoint=(wait=0,name=hi)")
         self.conn.reconfigure("checkpoint=(wait=5,name=hi)")
 
-    def test_reconfig_stat_log(self):
+    # Statistics logging: reconfigure the things we can reconfigure.
+    def test_reconfig_statistics_log_ok(self):
         self.conn.reconfigure("statistics=[all],statistics_log=(wait=0)")
-        self.conn.reconfigure("statistics_log=(wait=0)")
         self.conn.reconfigure("statistics_log=(wait=2)")
-        self.conn.reconfigure("statistics_log=(wait=0)")
-        self.conn.reconfigure("statistics_log=(wait=2,sources=[lsm:])")
-        self.conn.reconfigure("statistics_log=(wait=0)")
-        self.conn.reconfigure("statistics_log=(wait=2,timestamp=\"t%b %d\")")
-        self.conn.reconfigure("statistics_log=(wait=0)")
-        self.conn.reconfigure("statistics_log=(wait=2,path=\"wts.%d.%H\")")
-        self.conn.reconfigure("statistics_log=(wait=0)")
-        self.conn.reconfigure(
-             "statistics_log=(wait=2,sources=[lsm:],timestamp=\"%b\")")
+        self.conn.reconfigure("statistics_log=(wait=1,json=true)")
```

Comment:
The old code tested toggling statistics logging on and off, not just resetting different timer intervals. Is there a reason you took that out?